src/styles/stars.scss

```diff
@@ -0,0 +1,30 @@
+.rate {
+  padding-left: $padding-left;
+  margin-top: 16px;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+
+  &--stars {
+    display: flex;
+    &__star {
+      width: 16px;
+      height: 16px;
+      background-image: url(../images/star.svg);
+      background-repeat: no-repeat;
+      background-position: center;
+      margin-right: 4px;
+    }
+  }
+
+  &--stars-4 .rate--stars__star:nth-child(-n + 4) {
+    background-image: url(../images/star-active.svg);
+  }
+
+  &--reviews {
+    color: $text-color-strong;
```

Comment:
The variable `$text-color-strong` is used but not defined. Please define this variable to avoid compilation errors.